ballerina-tests/http-interceptor-tests/tests/interceptors_basic_tests.bal

```diff
@@ -602,6 +603,19 @@ function testJwtInformationInRequestContext() returns error? {
     test:assertEquals(reqCtxJwtValues[1]["scp"], "admin");
 }
 
+@test:Config{}
+function testEmptyJwtInformationInRequestContext() returns error? {
+    reqCtxJwtValues = [];
+    http:Client jwtClient = check new("https://localhost:" + jwtInformationInReqCtxtTestPort.toString(),
+    secureSocket = {
+        cert: common:CERT_FILE
+    });
+    http:Response response = check jwtClient->get("/requestInterceptorJwtInformation", {"authorization": "Basic "});
```

Comment:
I think this is not covering the fix. I think we are removing the trailing spaces when adding the request header